engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -158,11 +161,17 @@ public WorldRendererImpl(Context context, GLBufferPool bufferPool) {
         this.renderingConfig = context.get(Config.class).getRendering();
         this.renderingDebugConfig = renderingConfig.getDebug();
         this.shaderManager = context.get(ShaderManager.class);
-
-        if (renderingConfig.isOculusVrSupport()) {
-            playerCamera = new OculusStereoCamera();
-            currentRenderingStage = RenderingStage.LEFT_EYE;
-
+        if (renderingConfig.isVrSupport()) {
+            this.vrProvider = new OpenVRProvider();
+            if (this.vrProvider.init()) {
+                playerCamera = new OpenVRStereoCamera(this.vrProvider);
+                currentRenderingStage = RenderingStage.LEFT_EYE;
+                context.put(OpenVRProvider.class, vrProvider);
+            }
+            else {
```

Comment:
Fixed the formatting error.

if vrProvider fails to init, there is no scenario at this point where the log isn't blasted with errors, so I think we're safe to not add something here.